internal/pkg/runtime/launch/launcher_linux.go

```diff
@@ -1095,6 +1095,11 @@ func (l *Launcher) setCgroups(instanceName string) error {
 		return nil
 	}
 
+	if l.cfg.ShareNSMode {
+		sylog.Debugf("Instance stats will not be available because of sharens mode")
```

Comment:
This is just a debug message so not so important, but shouldn't it say that instance stats will not be available because cgroups v2 is not available?

I would also just put the if around the message, since the very next thing is the return nil anyway.

Reply:
I can see how you can get that impression from the messages printed, but the function is really about enabling cgroups.  Sharens mode itself is not the reason why instance stats won't work.  That's just the reason why we're turning the message from an Info message into a Debug message.